docs/user/field-selectors.md

```diff
@@ -0,0 +1,149 @@
+| **Array**        | `Contains`: Matches if the array contains the specified value.<br> `NotContains`: Matches if the array does not contain the specified value.<br> `In`: Matches if the array overlaps with the specified values.<br> `NotIn`: Matches if the array does not overlap with the specified values.<br> `Exists`: Matches if the field is present.<br> `DoesNotExist`: Matches if the field is not present.<br><br><i>**Note:** Using `Array[Index]` treats the element as the type defined for the array elements (e.g., string, timestamp, number, boolean).</i> | Array element    |
+| **JSON**        | `Equals`: Matches if the value at the specified key path is an exact match to the provided JSON value.<br> `DoubleEquals`: Matches if the value at the specified key path is an exact match to the provided JSON value (alternative to `Equals`).<br> `NotEquals`: Matches if the value at the specified key path does not match the provided JSON value. If the key path does not exist, it is considered a match.<br> `In`: Matches if the value at the specified key path matches at least one JSON value in the provided list.<br> `NotIn`: Matches if the value at the specified key path does not match any of the JSON values in the list. If the key path does not exist, it is considered a match.<br> `Exists`: Matches if the specified key path is present in the JSON field.<br> `DoesNotExist`: Matches if the specified key path is not present in the JSON field.<br> `Contains`: Matches if the value at the specified key path contains the provided JSON value.<br> `NotContains`: Matches if the value at the specified key path does not contain the provided JSON value. | JSON   |
+
+
+## Supported Fields
+
+The following table lists the fields supported for filtering Flight Control objects:
+
+| Field Name                   | Field Type           |
+|------------------------------|----------------------|
+| `metadata.name`              | `String`             |
+| `metadata.owner`             | `String`             |
+| `metadata.labels`            | `StringArray`        |
+| `metadata.annotations`       | `StringArray`        |
+| `metadata.creationTimestamp` | `Timestamp`          |
+| `metadata.deletionTimestamp` | `Timestamp`          |
+| `spec`                       | `JSON`               |
+| `status`                     | `JSON`               |
+
+
+### Usage Examples
+
+Use the `--field-selector` flag to filter devices based on field values. Here are some examples:
+
+#### Example 1: Exclude a Specific Device by Name
+This command filters out a specific device by its name:
+```bash
+flightctl get devices --field-selector 'metadata.name!=c3tkb18x9fw32fzx5l556n0p0dracwbl4uiojxu19g2'
+```
+#### Example 2: Filter by Owner, Labels, and Creation Timestamp
+This command retrieves devices owned by `Fleet/pos-fleet`, located in the `us` region, and created in 2024:
```

Comment:
```suggestion
This command retrieves devices owned by `Fleet/pos-fleet`, located in the `us` region, and created before 2025:
```

Reply:
but it also has 'metadata.creationTimestamp >= 2024-01-01T00:00:00Z'

Reply:
Sorry, for some reason I missed the check >= Jan 2024. Resolving comment :)